_projects/2024/100453237/100453237.Rmd

```diff
@@ -0,0 +1,589 @@
+---
+title: "Ternary graph by ‘The Economist’: The world’s most, and least, walkable cities"
```

Comment:
Just "The world’s most, and least, walkable cities" is better in my opinion. You could add that it's a ternary graph in the description below.